arduino_unit_tests/cc1120_ext_regs.h

```diff
@@ -0,0 +1,134 @@
+#ifndef EXT_REGS_CC1120_H
+#define EXT_REGS_CC1120_H
+
+#define IF_MIX_CFG          0x00
```

Comment:
C style way of writing these but if you do #define make sure you make them UNSIGNED (ex: 0x00U) to avoid weird errors